dle/update_database.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+echo `date`
+
+export PYTHONPATH=/usr/local/lib/python3.11/site-packages/
+export PATH=$PATH:/usr/local/bin/
+cd /app
+
+echo "Beginning the Django data ingest"
+
+echo "Loading EMA (EU) data"
+python3.11 manage.py load_ema_data --type full
+echo "Finished loading EMA (EU) data"
+
+echo "Loading FDA (US) data"
+python3.11 manage.py load_fda_data --type full
+echo "Finished loading FDA (US) data"
+
+echo "Loading TGA (Australia) data"
+python3.11 manage.py load_tga_data --type full
+echo "Finished loading TGA (Australia) data"
+echo "Updating latest drug labels"
+
+echo "Loading HC (Health Canada) data"
+python3.11 manage.py load_hc_data --type full
+echo "Finished loading HC (Health Canada) data"
+
+python3.11 manage.py update_latest_drug_labels
+echo "Finished updating latest drug labels"
+echo "Ended the Django data ingest"
+
+echo "Begin vectorizing labels"
+python3.11 manage.py vectorize --agency all
+echo "Done vectorizing labels"
+
+echo "Begin indexing to ES"
+python3.11 manage.py provision_elastic --agency all --create_index True --mapping_file "/app/search/mappings/provision.json"
```

Comment:
Working on this ... `create_index` should really be `recreate_index`, the index has to exist and the only time we really need confirmation through a flag is if we want to drop and recreate it. If the index exists and we don't have that flag, we ignore it and use the existing index (mappings stay the same, data stays). Because we are using the `productsection.id` from Django as the `doc._id` in Elastic, we can just load in all the data without worrying about dupes. If there is an `id` match, the document is updated (no dupes). This also means we don't need to worry about dropping and creating the index unless we want to update the mapping, and we can run an index while leaving the index available for searching (likely performance degradation but should still be ok).